bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Channel.java

```diff
@@ -168,4 +172,13 @@ public Configuration getConfiguration() {
     public Set<String> getDefaultTags() {
         return defaultTags;
     }
+
+    /**
+     * Returns the trigger type.
+     *
+     * @return trigger type
+     */
+    public String getTriggerType() {
```

Comment:
Maybe better move this next to getAcceptedItemType instead of the end of the file?